mobile/packages/swap/adapters/api/minswap/api-maker.ts

```diff
@@ -0,0 +1,325 @@
+              data: transformers.orders.response(response.value.data),
+            },
+          },
+          true,
+        )
+      },
+
+      /* istanbul ignore next */
+      async limitOptions({tokenIn, tokenOut}: Swap.LimitOptionsRequest) {
+        const estimateResponse = await this.estimate({
+          tokenIn,
+          tokenOut,
+          slippage: 0,
+          amountIn: 50,
+        })
+
+        if (isLeft(estimateResponse)) {
+          return estimateResponse
+        }
+
+        const wantedPrice = estimateResponse.value.data.netPrice
+        const defaultProtocol = estimateResponse.value.data.splits[0]?.protocol
+
+        if (defaultProtocol === undefined) {
+          return freeze<Left<Api.ResponseError>>(
+            {
+              tag: 'left',
+              error: {
+                status: -3,
+                message: 'Invalid state',
+                responseData: null,
+              },
+            },
+            true,
+          )
+        }
+
+        // Minswap only supports MinswapV2 protocol
+        const options = [
+          {
+            protocol: Swap.Protocol.Minswap_v2,
+            initialPrice: wantedPrice,
+            batcherFee: estimateResponse.value.data.batcherFee,
+          },
+        ]
+
+        const result = {
+          defaultProtocol,
+          wantedPrice,
+          options,
+        }
+
+        return freeze(
+          {
+            tag: 'right',
+            value: {
+              status: Api.HttpStatusCode.Ok,
+              data: result,
+            },
+          },
+          true,
+        )
+      },
+
+      async estimate(body: Swap.EstimateRequest) {
+        const requestBody = transformers.estimate.request(body)
+
+        const response = await requestWithErrorHandling<EstimateResponse>(
+          `${baseUrl}/estimate`,
+          {
+            method: 'POST',
+            body: JSON.stringify(requestBody),
+          },
+        )
+
+        if (isLeft(response)) return response
+
+        return freeze(
+          {
+            tag: 'right',
+            value: {
+              status: response.value.status,
+              data: transformers.estimate.response(response.value.data),
+            },
+          },
+          true,
+        )
+      },
+
+      async create(body: Swap.CreateRequest) {
+        const requestBody = transformers.create.request(body)
+
+        // Make the build-tx call
+        const response = await requestWithErrorHandling<CreateResponse>(
+          `${baseUrl}/build-tx`,
+          {
+            method: 'POST',
+            body: JSON.stringify(requestBody),
+          },
+        )
+
+        if (isLeft(response)) return response
+
+        // Make an ad-hoc estimate call to get the swap details
+        const estimateRequest: Swap.EstimateRequest = {
+          amountIn: body.amountIn,
+          tokenIn: body.tokenIn,
+          tokenOut: body.tokenOut,
+          slippage: body.slippage ?? 1,
```

Comment:
Yoroi allows 0% slippage.
```suggestion
          slippage: body.slippage ?? 0,
```